Kitodo-DataManagement/hibernate.properties

```diff
@@ -0,0 +1,3 @@
+hibernate.search.enabled=true
+hibernate.search.backend.hosts=localhost:9200
+hibernate.search.backend.protocol=http
```

Comment:
Question must this content not be added to the existing `hibernate.cfg.xml` file or did we need two configuration files?

Reply:
As I understand it, this is the configuration file for the Hibernate Search framework. I would be surprised if we could mix the two configurations.

Reply:
Does this mean, that everyone must run ElasticSearch / Opensearch on localhost and port 9200? If so I'm unable to do this in my development system nor on a productive environment.

Reply:
No, it does mean, if you run it elsewhere, you must change the host/port in the configuration file. This is at least my expectation, I did not try to change them, but that a configuration file should be for.